memor/params.py

```diff
@@ -86,9 +86,56 @@ class LLMModel(Enum):
     CLAUDE_2 = "claude-2"
     CLAUDE_INSTANT = "claude-instant"
 
-    LLAMA3_70B = "llama3-70b"
-    LLAMA3_8B = "llama3-8b"
+    LLAMA_2_7B = "llama-2-7b"
+    LLAMA_2_13B = "llama-2-13b"
+    LLAMA_2_70B = "llama-2-70b"
+    LLAMA_3_1_8B = "llama-3.1-8b"
+    LLAMA_3_1_8B_INSTRUCT = "llama-3.1-8b-instruct"
+    LLAMA_3_1_70B = "llama-3.1-70b"
+    LLAMA_3_1_70B_INSTRUCT = "llama-3.1-70b-instruct"
+    LLAMA_3_1_405B = "llama-3.1-405b"
+    LLAMA_3_1_405B_INSTRUCT = "llama-3.1-405b-instruct"
+    LLAMA_3_2_1B = "llama-3.2-1b"
+    LLAMA_3_2_1B_INSTRUCT = "llama-3.2-1b-instruct"
+    LLAMA_3_2_3B = "llama-3.2-3b"
+    LLAMA_3_2_3B_INSTRUCT = "llama-3.2-3b-instruct"
+    LLAMA_3_2_11B = "llama-3.2-11b"
+    LLAMA_3_2_11B_INSTRUCT = "llama-3.2-11b-instruct"
+    LLAMA_3_2_11B_VISION_INSTRUCT = "llama-3.2-11b-vision-instruct"
+    LLAMA_3_2_90B = "llama-3.2-90b"
+    LLAMA_3_2_90B_INSTRUCT = "llama-3.2-90b-instruct"
+    LLAMA_3_2_90B_VISION = "llama-3.2-90b-vision"
+    LLAMA_3_2_90B_VISION_INSTRUCT = "llama-3.2-90b-vision-instruct"
+    LLAMA_3_3_70B = "llama-3.3-70b"
+    LLAMA_3_3_70B_INSTRUCT = "llama-3.3-70b-instruct"
+    LLAMA_4_MAVERICK_17B_128E = "llama-4-maverick-17b-128e"
+    LLAMA_4_MAVERICK_17B_128E_INSTRUCT = "llama-4-maverick-17b-128e-instruct"
+    LLAMA_4_MAVERICK_17B_16E = "llama-4-maverick-17b-16e"
+    LLAMA_4_MAVERICK_17B_16E_INSTRUCT = "llama-4-maverick-17b-16e-instruct"
+    LLAMA_GUARD_7B = "llama-guard-7b"
+    LLAMA_GUARD_3_1B = "llama-guard-3-1b"
     LLAMA_GUARD_3_8B = "llama-guard-3-8b"
+    LLAMA_GUARD_3_11B_VISION = "llama-guard-3-11b-vision"
+    LLAMA_GUARD_4_12B = "llama-guard-4-12b"
+    LLAMA_PROMPT_GUARD_2_22M = "llama-prompt-guard-2-22m"
+    LLAMA_PROMPT_GUARD_2_86M = "llama-prompt-guard-2-86m"
+    LLAMA_4_SCOUT_17B_128E = "llama-4-scout-17b-128e"
+    LLAMA_4_SCOUT_17B_16E = "llama-4-scout-17b-16e"
+    META_LLAMA_3_8B = "meta-llama-3-8b"
```

Comment:
I believe `META_LLAMA_3` is equivalent to `LLAMA_3`, so I suggest converting these model names to `LLAMA_3_xx`.